### PR TITLE
fix #237: [FR] Sub sub list not handled correctly

### LINKED ISSUE #237: [FR] Sub sub list not handled correctly
- Wiktionary page: https://fr.wiktionary.org/w/index.php?title=marron

Wikicode:
```
# {{par analogie|fr}}
## {{term|de couleur}}
### [[brun|Brun]], couleur du fruit [[mûr]]. 
```

Output:
```
3. (Par analogie)
      a. (De couleur)
```

Expected:
```
2. (Par analogie)
      a. (De couleur)
            i. Brun, couleur du fruit mûr
```

---

Model link, if any:

### PR DESCRIPTION
**Pull request by lasconic**
- [x] Fixes #
- [x] Tests added/updated.
- [x] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

**Comment by lasconic** [comment]
One more level and it reaches my limit for a function... What's yours ?

File: scripts/stubs.py [hunk 1]
```diff
@@ -2,7 +2,8 @@
 from collections import namedtuple
 from typing import Dict, Tuple, Union
 
-Definitions = Union[str, Tuple[str, ...]]
+Subdefinitions = Union[str, Tuple[str, ...]]
```
Comment (BoboTiG) [inline scripts/stubs.py:5]: Do you mind chnging the case to `SubDefinitions`?

Comment (lasconic) [reply scripts/stubs.py]: No pb

File: scripts/get.py [hunk 2]
```diff
@@ -356,12 +367,20 @@ def strip_html(text: str) -> str:
...
 
-    for i, definition in enumerate(details.definitions, start=1):
+    index = 1
+    for definition in details.definitions:
         if isinstance(definition, tuple):
             for a, subdef in zip("abcdefghijklmopqrstuvwxz", definition):
-                print(f"{a}.".rjust(8), strip_html(subdef))
+                if isinstance(subdef, tuple):
+                    for rn, subsubdef in zip(
+                        ["i", "ii", "iii", "iv", "v", "vi", "vii"], subdef
```
Comment (BoboTiG) [inline scripts/get.py:376]: Hopefully 7 is the maximum haha
Good to go as-is 👍

**Review by BoboTiG — APPROVED** [review]

**Pull request merged**